sdk/tools/azure-openrewrite/src/main/resources/META-INF/rewrite/migrationRecipes/core/com.azure.core.client.traits.yml

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------
+# Migration Recipes for azure.core.clients.traits
+# --------------------------------------------------------------------
+type: specs.openrewrite.org/v1beta/recipe
+name: com.azure.openrewrite.migration.azure.core.clients.traits
+displayName: Migrate from azure.core.clients.traits to next generation stack
+description: This recipe migrates azure.core.clients.traits package to the next generation stack.
+recipeList:
+
+  # --------------------------------------------------------------------
+  # AzureKeyCredentialTrait
+  # --------------------------------------------------------------------
+
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: com.azure.core.client.traits.AzureKeyCredentialTrait
+      newFullyQualifiedTypeName: com.azure.v2.core.traits.AzureKeyCredentialTrait
```

Comment:
Same here - these traits are not expected to be used in user applications. Why do we need these recipes?

Reply:
These were enthusiastically added. We've determined we won't need recipes for these apis and therefore these will be removed in a cleanup PR.